### ISSUE TITLE
Activity bar icon focus indicator is hard to see (probably should be independently themable)

### ISSUE THREAD

**Issue by eamodio** [microsoft/vscode#83328]
If you switch to the explorer sidebar when there is no folder/workspace open you will see the active activity bar line is not correct.
![image](https://user-images.githubusercontent.com/641685/67597427-2657de00-f739-11e9-9346-cde7d6a4c845.png)

This seems to happen anytime a sidebar doesn't set input focus into the view. It also can happen with custom views.

**Comment by misolori** [comment]
This is by design, when you click on the file explorer and there is nothing in the sidebar the focus is changed to the activity bar item. So if you hit "tab" you will start tabbing through the activity bar. We hide the active border when an activity bar item is focused.

![image](https://user-images.githubusercontent.com/35271042/67604076-77fb6b00-f72f-11e9-90fa-9e0a645a39e0.png)

**Comment by misolori** [comment]
cc @isidorn for focus in the activity bar items

**Comment by eamodio** [comment]
After talking with @misolori I am re-opening this, but changing it to be specifically about the focus indicator color. And will be opening another issue about the view focus strangeness.

**Comment by misolori** [comment]
Based on conversations, I'll introduce a new color token for the active border that will be something like `activityBar.activeFocusBorder` for when the item is in focus.

**Comment by misolori** [comment]
You can theme the active border when it is focused via `activityBar.activeFocusBorder`:

<img src="https://user-images.githubusercontent.com/35271042/68715559-28060c00-0557-11ea-89cd-00b2424bc085.gif">
